library/pickle.po

```diff
@@ -6,21 +6,22 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-06-10 11:27+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2018-12-06 22:27+0100\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 2.0.6\n"
 
 #: ../Doc/library/pickle.rst:2
 msgid ":mod:`pickle` --- Python object serialization"
-msgstr ""
+msgstr ":mod:`pickle` --- Module Python de sérialisation en objet"
 
 #: ../Doc/library/pickle.rst:10
 msgid "**Source code:** :source:`Lib/pickle.py`"
-msgstr ""
+msgstr "**Source code:** :source:`Lib/pickle.py`"
```

Comment:
Unchanged ?
```suggestion
msgstr "**Code source :** :source:`Lib/pickle.py`"
```

Reply:
Prise en main de l'outil toujours (je pensais que vous pouviez faire les modifs directement)
J'essaie de m'y mettre demain

Reply:
@MikaelMonjour du coup tu peux accepter directement dans github les propositions de modifications que on fait. CF https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request
😄